### ISSUE TITLE
Mincraft no listing

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1016]
No listing of price for minecraft except for amazon.

[Issue created by BoyRandom: 2018-05-12]
[Last updated on bitbucket: 2019-07-23]

[Comment created by Knightw0lf: 2019-07-23]
it pulls now from   
Amazon, humble bundle, \(java\)  
Minecraft for Windows 10 version \(bedrock\)  
Minecraft: New Nintendo 3DS Edition \(bedrock\)  
but if any storefront does support it, it will appear later  
this can be closed

[Comment created by Dr_Gore: 2018-05-28]
This is because ITAD automatically pulls data from select stores. The reason that you cannot find Minecraft listed anywhere other than Amazon is that Amazon is the only seller that both sells Minecraft AND is tracked by ITAD.

So, for example, Minecraft is not sold on Steam, Humble Store, GOG, etc. you can find a full list of storefronts that ITAD tracks here: https://isthereanydeal.com/status/
Where it IS sold is on Minecraft's very on storefront and website. This makes it unreasonable for the makers of ITAD to try and track as to track it they would have to start tracking the storefront for Minecraft which only contains one game.

TLDR; Minecraft isn't sold on a supported storefront and it'd be unreasonable for ITAD to try and track independent sellers across the web.

**Comment by MarkitkaM** [comment]
https://isthereanydeal.com/game/minecraft/info/
Amazone + Humble store
